repl/src/main/scala/ammonite/repl/interp/ClassLoaders.scala

```diff
@@ -121,11 +123,42 @@ class SpecialClassLoader(parent: ClassLoader, parentHash: Array[Byte])
     * classloader can get at them.
     */
   val newFileDict = mutable.Map.empty[String, Array[Byte]]
+  val classOutputDir = System.getProperty("ammonite.repl.class.outputdir", null)
```

Comment:
Can we inject this as a parameter from `new Repl(...)`, instead of relying on a global property here?